static/styles/user_status.css

```diff
@@ -1,58 +1,11 @@
-.user_status_overlay {
-    .overlay-content {
-        width: 384px;
```

Comment:
I restored this 384px width, since otherwise the modal looks really weirdly designed.  

(The intent was to replace the infrastructure, not to change the design).